From 25c9cc834749f5f564cc8d74c44ec6db31302ce5 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Tue, 11 Feb 2020 17:07:28 -0600
Subject: [PATCH 09/21] arm64: dts: beacon-imx8mn-kit: Remove dead DTS entries

The beacon-imx8mn-kit was based on the Mini version, however,
the Nano doesn't support USBOTG2 nor does it support PCIe.

This patch removes legacy references to the regulators which
power these peripherals on the Mini but are not applicable to
the Nano version.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 .../freescale/beacon-imx8mn-baseboard.dtsi    | 38 -------------------
 1 file changed, 38 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi
index 627861aa04db..bebeaba8e752 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi
@@ -33,25 +33,6 @@
 		};
 	};
 
-	reg_pcie_pwr_en: regulator-pci_pwr_en {
-		compatible = "regulator-fixed";
-		regulator-name = "pci_pwr_en";
-		regulator-min-microvolt = <3300000>;
-		regulator-max-microvolt = <3300000>;
-		enable-active-high;
-		gpio = <&pca6416_1 1 GPIO_ACTIVE_HIGH>;
-		vin-supply = <&reg_pcie_nclkreq>;
-		startup-delay-us = <5000000>;
-	};
-
-	reg_pcie_nclkreq: regulator-pcie_nclkreq {
-		compatible = "regulator-fixed";
-		regulator-name = "pcie_nclkreq";
-		regulator-min-microvolt = <3300000>;
-		regulator-max-microvolt = <3300000>;
-		gpio = <&pca6416_1 2 GPIO_ACTIVE_LOW>;
-	};
-
 	reg_usdhc2_vmmc: regulator-usdhc2 {
 		compatible = "regulator-fixed";
 		regulator-name = "VSD_3V3";
@@ -73,15 +54,6 @@
 		enable-active-high;
 	};
 
-	reg_usb_h1_vbus: regulator-usbh1vbus {
-		compatible = "regulator-fixed";
-		regulator-name = "usb_h1_vbus";
-		regulator-min-microvolt = <5000000>;
-		regulator-max-microvolt = <5000000>;
-		gpio = <&pca6416_1 7 GPIO_ACTIVE_LOW>; /* nRST */
-		enable-active-low;
-	};
-
 	reg_audio: regulator-audio {
 		compatible = "regulator-fixed";
 		regulator-name = "3v3_aud";
@@ -167,7 +139,6 @@
 			>;
 		};
 
-
 		pinctrl_pdm: pdmgrp {
 			fsl,pins = <
 				MX8MN_IOMUXC_SAI5_MCLK_SAI5_MCLK	0xd6
@@ -180,13 +151,6 @@
 			>;
 		};
 
-		pinctrl_pcie0: pcie0grp {
-			fsl,pins = <
-				MX8MN_IOMUXC_GPIO1_IO05_GPIO1_IO5	0x41
-				MX8MN_IOMUXC_SAI2_RXFS_GPIO4_IO21	0x41
-			>;
-		};
-
 		pinctrl_reg_usb_otg: reg_usb_otg {
 			fsl,pins = <
 				MX8MN_IOMUXC_SAI3_RXC_GPIO4_IO29     0x19
@@ -333,7 +297,6 @@
 			};
 		};
 	};
-
 };
 
 &micfil {
@@ -361,7 +324,6 @@
 };
 
 &i2c4 {
-
 	clock-frequency = <400000>;
 	pinctrl-names = "default", "gpio";
 	pinctrl-0 = <&pinctrl_i2c4>;
-- 
2.17.1

